test_conformance/events/test_event_dependencies.cpp

```diff
@@ -275,6 +276,11 @@ int test_event_enqueue_wait_for_events_run_test(
 
     for (loop_count = 0; loop_count < TEST_COUNT; loop_count++)
     {
+        int event_count = 0;
+        clEventWrapper first_dependency = (loop_count == 0) ? pre_loop_event : last_loop_event;
```

Comment:
This needs a [clang-format to pass CI](https://github.com/KhronosGroup/OpenCL-CTS/actions/runs/4512804381/jobs/7978464519?pr=1678)

```diff
diff --git a/test_conformance/events/test_event_dependencies.cpp b/test_conformance/events/test_event_dependencies.cpp
index cd0aa67..e94c8cc 100644
--- a/test_conformance/events/test_event_dependencies.cpp
+++ b/test_conformance/events/test_event_dependencies.cpp
@@ -277,8 +277,10 @@ int test_event_enqueue_wait_for_events_run_test(
     for (loop_count = 0; loop_count < TEST_COUNT; loop_count++)
     {
         int event_count = 0;
-        clEventWrapper first_dependency = (loop_count == 0) ? pre_loop_event : last_loop_event;
-        clEventWrapper event[SIZE]; // If we usemarkers we get 2 more events per iteration
+        clEventWrapper first_dependency =
+            (loop_count == 0) ? pre_loop_event : last_loop_event;
+        clEventWrapper
+            event[SIZE]; // If we usemarkers we get 2 more events per iteration
         event[event_count] = first_dependency;
 
         // Execute kernel 1
```

Reply:
done , thank you !!